ParseFacebookUtils/ParseFacebookUtils.xcodeproj/project.pbxproj

```diff
@@ -9,8 +9,6 @@
 /* Begin PBXBuildFile section */
 		4A0ECC80200D99FF00BA84A3 /* Bolts.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 4AAEAA45200BE26C00AA7479 /* Bolts.framework */; };
 		4A0ECC84200D9A0D00BA84A3 /* Bolts.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 4AAEAA45200BE26C00AA7479 /* Bolts.framework */; };
-		4A0ECC88200D9A1D00BA84A3 /* Parse.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 4A9A9503200D1462005D8F4B /* Parse.framework */; };
```

Comment:
I've isolated that removing these causes "carthage update" to fail earlier than normal. Can I ask what was the purpose of removing them?

Reply:
I think it was me trying to understand the dependency setup in the project, and not seeing that this one did anything; the product built and tested fine in Xcode and from the CLI without this, so I assumed it could be pruned.

By the looks of things, this assumption was incorrect, since you're running into issues using the product from Carthage in a client product.

Reply:
Ahh... you're right about https://github.com/parse-community/Parse-SDK-iOS-OSX/commit/4a820993149ec524cf45d3d6700143399ca2d5ac. That's why I made that change. I assumed that we'd inherit that change when this PR was merged into master.

Sorry... it's been long enough since I looked at this that I can't quite remember how it played out :)

Reply:
Actually... maybe it's because it's early-ish on Sunday here, but I can't see the static / dynamic separation rollback in that commit. It's here: https://github.com/parse-community/xctoolchain/commit/690787abfa21eea11095adf662f6e71154b3b0c8, and that submodule update is currently part of #1428.

Reply:
@drdaz you’re right, I referenced the wrong commit. Either way my experimentation shows that removing these framework references causes “Carthage update” to fail. Can we put them back?

Reply:
It sounds like it needed to be there, so definitely.

I confess I didn't test building the project as a dependency using Carthage before pushing those changes. Need to remember to do that I guess.

Reply:
Well this is fun... I rolled back the removal of Parse.framework and pushed it to master in rico's repo. The commit isn't included here. I thought that was supposed to happen automagically?